### PR TITLE
Bug fixed for org avatar caused by #1049

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

File: models/user.go [hunk 1]
```diff
@@ -324,12 +324,12 @@ func (u *User) RelAvatarLink() string {
 
 	switch {
 	case u.UseCustomAvatar:
-		if !com.IsExist(u.CustomAvatarPath()) {
+		if !com.IsExist(u.CustomAvatarPath()) || !com.IsFile(u.CustomAvatarPath()) {
```
Comment (ethantkoenig) [inline models/user.go]: Can just have `!com.IsFile(u.CustomAvatarPath())`; `com.IsFile(..)` will return `false` if the file does not exist.

**Review by ethantkoenig — CHANGES_REQUESTED** [review]

**Comment by ethantkoenig** [comment]
One small tweak, otherwise LGTM

**Review by ethantkoenig — APPROVED** [review]
LGTM

**Comment by lunny** [comment]
let L-G-T-M work

**Pull request merged**